docker-bake.hcl

```diff
@@ -138,6 +143,20 @@ target "all-cross" {
   inherits = ["all", "_platforms"]
 }
 
+#
+# bin image
+#
+
+target "bin-image" {
```

Comment:
This one is for building locally for the current platform right?

Reply:
Yes